week2/materials/homework/assignments.md

```diff
@@ -4,51 +4,86 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `http://jean-claude-week2.s3-website-us-east-1.amazonaws.com`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
+service name: `AWS Certificate Manager (ACM)`
 
 **Assignment 6:**
 
 Figure out and write down the price per month of storing 51TB on S3.
 
-Price: `?`
+Price: `7385.63DKK`
 
-Total size of website: `?`
+Total size of website: `494.4kb`
+![size!](images/websiteSize.PNG)
```

Comment:
Very useful for me to see the screenshots! 👍

Reply:
Thanks